packages/algob/test/lib/tx.ts

```diff
@@ -70,6 +70,8 @@ describe("Opt-In to ASA", () => {
     };
     sinon.stub(algod.algodClient, "getTransactionParams")
       .returns({ do: async () => mockSuggestedParam } as ReturnType<algosdk.Algodv2['getTransactionParams']>);
+    sinon.stub(algod.algodClient, "genesis")
+      .returns({ do: async () => mockGenesisInfo } as ReturnType<algosdk.Algodv2['genesis']>);
```

Comment:
this is repeated, shall we put it into a function? Eg: `stubAlgodGenesis(algod)`?

Reply:
yes